src/app/(main)/my-voyage/[teamId]/tech-stack/components/TechStackCard.tsx

```diff
@@ -23,15 +32,83 @@ export default function TechStackCard({ title, data }: TechStackCardProps) {
   const inputRef = useRef<HTMLInputElement>(null);
   const editRef = useRef<HTMLInputElement>(null);
   const items = data.map((item) => item.name.toLowerCase());
+  const params = useParams<{ teamId: string }>();
+  const teamId = Number(params.teamId);
   const userId = useUser().id;
+
+  //voyageTeamMembers, currentTeam and voyageTeamMemberId are all used to get
+  //TODO: create hook or find simpler more readble way to get voyageTeamMemberId?
+  const voyageTeamMembers = useAppSelector(
+    (state) => state.user?.voyageTeamMembers || [],
+  );
+  const currentTeam = useMemo(
+    () =>
+      voyageTeamMembers.filter(
+        (team) =>
+          team.voyageTeam.voyage.status.name === "Active" &&
+          team.voyageTeamId === teamId,
+      ),
+    [voyageTeamMembers, teamId],
+  );
+  const voyageTeamMemberId = currentTeam.length > 0 ? currentTeam[0].id : -1;
```

Comment:
There is logic for this already that you can use in the getcurrentvoyageteam utils file